src/EditorFeatures/Core.Wpf/Adornments/AbstractAdornmentManager.cs

```diff
@@ -251,16 +252,26 @@ protected void UpdateSpans_CallOnlyOnUIThread(NormalizedSnapshotSpanCollection c
             AddAdornmentsToAdornmentLayer_CallOnlyOnUIThread(changedSpanCollection);
         }
 
-        protected bool ShouldDrawTag(SnapshotSpan snapshotSpan, IMappingTagSpan<T> mappingTagSpan, out SnapshotPoint mappedPoint)
+        protected bool ShouldDrawTag(SnapshotSpan snapshotSpan, IMappingTagSpan<T> mappingTagSpan, out IWpfTextViewLine viewLine)
         {
-            mappedPoint = default;
+            viewLine = null;
             var point = GetMappedPoint(snapshotSpan, mappingTagSpan);
 
             if (point is null)
             {
                 return false;
             }
 
+            var mappedPoint = point.Value;
+            viewLine = TextView.TextViewLines.GetTextViewLineContainingBufferPosition(mappedPoint);
+
+            // Unsure what the scenario is - but sometimes the SnapshotPoint is not located
+            // within any of the lines in the TextViewLineCollection. In that case, we do not want to draw a tag.
+            if (viewLine is null)
+            {
+                return false;
+            }
```

Comment:
did you validate that this line-separates and raw-string-indentation-tags still show up as expected?

Reply:
Yep, they seem to show up the same as they did before